src/Database/Driver/sparql/Connection.php

```diff
@@ -168,8 +166,7 @@ public function getLogger(): ?Log {
    * {@inheritdoc}
    */
   public static function open(array &$connection_options = []): Client {
-    // @todo Get endpoint string from settings file.
-    $connect_string = 'http://' . $connection_options['host'] . ':' . $connection_options['port'] . '/sparql';
+    $connect_string = "http://{$connection_options['host']}:{$connection_options['port']}/sparql";
```

Comment:
This is out of scope for this PR, but shouldn't we allow HTTPS for database connections? Seems a quite critical feature to me, communicating with a remote database server over HTTPS is a very common use case.

Reply:
Yes, right. It's true also for the path for which we could use the database connection key. Will handle this.